lib/Common/CommonPal.h

```diff
@@ -621,7 +632,7 @@ namespace PlatformAgnostic
 {
     __forceinline unsigned char _BitTestAndSet(LONG *_BitBase, int _BitPos)
     {
-#if defined(__clang__)
+#if defined(__clang__) && !defined(_ARM_)
```

Comment:
My ARM fu is weak, so a basic question here- is _ARM_ the same for Windows ARM vs Android ARM v7? Should it be? I know that Windows supports Thumb2, which I don't know if Android supports so the Windows ARM code might end up being slightly different from the Android ARM- I'm not sure what the macro model here should look like. @pleath @satheeshravi thoughts?